src/actions/copy-environment/index.ts

```diff
@@ -23,9 +22,10 @@ export async function main(): Promise<void> {
     const parameterMap = taskParser.getHostParameterEntries(runnerParameters.workingDir, "copy-environment");
 
     core.startGroup('copy-environment:');
+    const sourceUrl = core.getInput('source-url', { required: true});
     await copyEnvironment({
         credentials: getCredentials(),
-        sourceEnvironmentUrl: getEnvironmentUrl(),
+        sourceEnvironmentUrl: sourceUrl,
```

Comment:
why not use the parameterMap as we already have for target-url?

Reply:
source url is expecting a string at cli-wrapper level, if we want to use parameter map then I need to update source env url type to string | HostParameterEntry